src/Pages/Product/Product.jsx

```diff
@@ -0,0 +1,134 @@
+import Header from "../../Components/Header/Header";
+import product from "../../Images/product.png";
+import styles from "./Product.module.css";
+const Product = () => {
+    function handleRange(e){
+        console.log(e.target.value);
+    }
+    return (
+        <>
+        <Header />
+        <div className="container">
+            <aside className={styles.filter_panel}>
+            <div className={styles.filter_option}>
+                <h3>Filters</h3>
+                <button className={styles.clear_btn}>Clear</button>
+            </div>
+            <div className="price-option">
+                <h3>Price</h3>
+                <div className={styles.price_value}>
+                    <span>1k</span>
+                    <span>2k</span>
+                    <span>3k</span>
+                    <span>4k</span>
+                    <span>5k</span>
+                </div>
+                <input type="range" name="price_range" step={1000} min={1000} max={5000} onInput={handleRange}/>
+            </div>
+            <div className="category-option">
+                <h3>Category</h3>
+                <div className="input-option">
+                    <input type="checkbox" name="men" id="men"/>
+                    <label htmlFor="men">Men Clothing</label>
+                </div>
+                <div className="input-option">
+                    <input type="checkbox" name="women" id="women"/>
+                    <label htmlFor="women">women Clothing</label>
+                </div>
+            </div>
```

Comment:
Also have your filters as a separate components.
And inside that also try to have components for each filter and then use them in filters.
Later on when you will use reducer then you code will become very junky so try to divide them in components from now only.